plugins/out_opensearch/os_conf.c

```diff
@@ -146,6 +157,8 @@ struct flb_opensearch *flb_os_conf_create(struct flb_output_instance *ins,
         snprintf(ctx->uri, sizeof(ctx->uri) - 1, "%s/_bulk", path);
     }
 
+    snprintf(ctx->template_uri, sizeof(ctx->template_uri) - 1, "%s/_index_template/%s", path, ctx->data_stream_template_name);
+    snprintf(ctx->ds_uri, sizeof(ctx->ds_uri) - 1, "%s/_data_stream/%s", path, ctx->data_stream_name);
```

Comment:
What is caused by `NULL` where are ctx->data_stream_template_name and ctx->data_stream_name?
This string operations should be proceeded only if they are specified or enabled data stream mode. What do you think?

Reply:
By default data_stream_name will be set to `fluent-bit` and data_stream_template_name will be `fluent-bit_template` when they're not provided. Do you think I should explicitly handle this? Something like:
```
if (ctx->data_stream_mode == FLB_TRUE && ctx->data_stream_name != NULL && ctx->data_stream_template_name != NULL)
...
```
My thinking was that the defaults for these values are non-null so we should be good in any case. Let me know if I should add handling here.

Reply:
Sorry, my bad. Your patch is correct.